AK/ByteBuffer.h

```diff
@@ -153,11 +153,6 @@ class ByteBuffer {
         m_size = 0;
     }
 
-    ALWAYS_INLINE void resize(size_t new_size)
-    {
-        MUST(try_resize(new_size));
-    }
-
```

Comment:
Eh
This feels pointless, it's the same thing with the callsite getting uglier.
Maybe renaming the function to something like `resize_but_fixme_should_propagate_errors()` or whatever the other APIs named like that are would be better?